web/modules/custom/ec_shortcodes/src/Plugin/EmbeddedContent/ECShortcodesFeaturedResource.php

```diff
@@ -103,7 +103,7 @@ public function buildConfigurationForm(array $form, FormStateInterface $form_sta
       '#required' => TRUE,
       '#selection_settings' => [
         'target_bundles' => [
-          'authors', 'basic_page', 'community', 'event', 'guide',
+          'authors', 'basic_page', 'community', 'event', 'guides',
           'news', 'resources', 'services', 'topics',
```

Comment:
is there a services type that makes sense here?  now I'm wondering if short_posts should be here too...